app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```diff
@@ -0,0 +1,89 @@
+@import "mixins/grid-helper";
+@import "mixins/prefixed-transform";
+
+.gem-c-cards__heading {
+  margin: 0 0 govuk-spacing(6) 0;
+}
+
+.gem-c-cards__list {
+  @include columns($items: 16, $columns: 1, $selector: "li");
```

Comment:
This relates to the issue I raised with our team this morning about the number of items being fixed to support IE11 which Ian has since mentioned [here](https://github.com/alphagov/govuk_publishing_components/pull/2624#pullrequestreview-889623142). 

I'll have a think about the best way forward.